resolwe/flow/managers/base.py

```diff
@@ -92,80 +90,49 @@ def communicate(self, run_sync=False, verbosity=1):
                         data.save()
                         continue
 
-                    script = self.exprengines['dtlbash'].eval(data)
-                    if script is None:
+                    try:
+                        execution_engine = data.process.run.get('language', 'bash')
+                        # TODO: Make it an error if there is no explicit language.
+                        program = self.get_execution_engine(execution_engine).evaluate(data)
```

Comment:
What happens when an invalid execution engine (say `python`) is defined in the process? Write a test and handle the error.